### PR TITLE
Update description of environment roles

### PR DESCRIPTION
**Pull request by sweetsweetstefan**
EN, FR, ES

### CODE REVIEW

File: fr/administration/rbac.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Tous les rôles système et rôles personnalisés dans l'organisation sont énum
 Pour contrôler l'accès aux ressources dans un environnement, CloudMC conçoit le *rôle d'environnement*.  En ajouter un membre nouveau à un environnement, il faut assigner un rôle d'environnment à l'utilisateur, qui régit le niveau d'accès accordé à l'utilisateur à l'intérieur de l'environnement.  La plupart des modules d'extension sont livrés avec le rôles d'environnment standards celles-ci :
 
 - **Observateur :**  Accès en lecture seule à l'environnement.
-- **Éditeur :**  Peut modifier toutes les charactéristiques de l'environnement.  Pas capable ni de changer les paramètres de l'evironnment ni de gérer les utilisateurs.
+- **Éditeur :**  Peut lire, écrire, et provisionner des ressources dans l'environnement.  Pas capable ni de changer les paramètres de l'evironnment ni de gérer les utilisateurs.
```
Comment (sweetsweetstefan) [inline fr/administration/rbac.md:75]:
```suggestion
- **Éditeur :**  Peut lire, écrire, et provisionner des ressources dans l'environnement, mais ne peut pas modifier les paramètres de l'environnment ni de gérer les utilisateurs.
```

File: fr/administration/rbac.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Tous les rôles système et rôles personnalisés dans l'organisation sont énum
 Pour contrôler l'accès aux ressources dans un environnement, CloudMC conçoit le *rôle d'environnement*.  En ajouter un membre nouveau à un environnement, il faut assigner un rôle d'environnment à l'utilisateur, qui régit le niveau d'accès accordé à l'utilisateur à l'intérieur de l'environnement.  La plupart des modules d'extension sont livrés avec le rôles d'environnment standards celles-ci :
 
 - **Observateur :**  Accès en lecture seule à l'environnement.
-- **Éditeur :**  Peut modifier toutes les charactéristiques de l'environnement.  Pas capable ni de changer les paramètres de l'evironnment ni de gérer les utilisateurs.
+- **Éditeur :**  Peut lire, écrire, et provisionner des ressources dans l'environnement, mais ne peut pas modifier les paramètres de l'environnment ni de gérer les utilisateurs.
```
Comment (joelplourde4) [inline fr/administration/rbac.md:75]:
```suggestion
- **Éditeur :**  Peut lire, écrire, et provisionner des ressources dans l'environnement, mais ne peut pas modifier les paramètres de l'environnement ni de gérer les utilisateurs.
```

**Review by joelplourde4 — COMMENTED** [review]
Little typo in French

File: fr/administration/rbac.md (hunk 2 above)
Comment (razine-bensari) [inline fr/administration/rbac.md:75]:
```suggestion
- **Éditeur :**  Peut lire, écrire, et provisionner des ressources dans l'environnement, mais ne peut pas modifier les paramètres de l'environnement ni de gérer les utilisateurs.
```

**Review by razine-bensari — APPROVED** [review]
FR LGTM, (one small typo for environnement)

File: fr/administration/rbac.md (hunk 2 above)
Comment (razine-bensari) [reply fr/administration/rbac.md:75]: you beat me by 17 seconds!

Comment (sweetsweetstefan) [reply fr/administration/rbac.md]: e775800

Comment (sweetsweetstefan) [reply fr/administration/rbac.md]: lolllll you guys

Comment (sweetsweetstefan) [reply fr/administration/rbac.md]: e775800

**Pull request merged**